dockerfiles/theia-endpoint-runtime/Dockerfile

```diff
@@ -8,54 +8,26 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia-dev:${BUILD_TAG} as builder
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia:${BUILD_TAG} as builder
 
+USER root
 
-# define in env variable GITHUB_TOKEN only if it is defined
-# else check if github rate limit is enough, else will abort requiring to set GITHUB_TOKEN value
-ARG GITHUB_TOKEN
+RUN cd /home/theia && \
+    yarn global add nexe@3.3.2 && nexe -v && \
+    # Build remote binary with node runtime 10.14.2 and che-theia node dependencies. nexe icludes to the binary only
+    # necessary dependencies.
+    nexe node_modules/@eclipse-che/theia-remote/lib/node/plugin-remote.js -t alpine-x64-10.14.2 -o /plugin-remote
 
-# Check github limit
-RUN if [ ! -z "${GITHUB_TOKEN-}" ]; then \
-        export GITHUB_TOKEN=$GITHUB_TOKEN; \
-        echo "Setting GITHUB_TOKEN value as provided"; \
-    else \
-        export GITHUB_LIMIT=$(curl -s 'https://api.github.com/rate_limit' | jq '.rate .remaining'); \
-        echo "Current API rate limit https://api.github.com is ${GITHUB_LIMIT}"; \
-        if [ "${GITHUB_LIMIT}" -lt 10 ]; then \
-            printf "\033[0;31m\n\n\nRate limit on https://api.github.com is reached so in order to build this image, "; \
-            printf "the build argument GITHUB_TOKEN needs to be provided so build will not fail.\n\n\n\033[0m"; \
-            exit 1; \
-        else \
-            echo "GITHUB_TOKEN variable is not set but https://api.github.com rate limit has enough slots"; \
-        fi \
-    fi
+# Light image without node. We include remote binary to this image.
+FROM alpine
```

Comment:
hello, please add the version for the image (to be sure to not get unwanted) updates